cypress/e2e/apitest/customers_request.cy.js

```diff
@@ -0,0 +1,100 @@
+/// <reference types="cypress" />
+
+
+
+describe('All test intermediary API',() => {
+    context('Request all customers', () => {
+        beforeEach('Get request', () => {
+            cy.request(Cypress.env('baseUrl')).as('customers')
+        })
+        it('Check status code return for all customers', () => {
+            cy.get('@customers').then((response) =>{
+                expect(response.status).to.eq(200)
+            })
+        })
+        it('Check length for return with all customers', () => {
+            cy.get('@customers').then((response) =>{
+                expect(response.body.customers).to.length(10)
+            })
+        })
+    })
+    context('Request all customers with pagination', () => {
+        const pagination = [1,2,3,4,5,6,7]
+        it.each(pagination)('Check pagination with all customers', (page) => {
+            cy.request(`${Cypress.env('baseUrl')}?page=${page}`).should((response) => {
+                expect(response.status).to.eq(200)
+                expect(response.body.pageInfo.currentPage).to.eq(page)
+            })
+        })
+    })
+    context('Request with get limit customer', () =>{
+        const limit = [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30]
+        it.each(limit)('Filter to customers with inside limit', (limit) => {
+            cy.request(`${Cypress.env('baseUrl')}?page=1&limit=${limit}`).then((response) => { 
+                expect(response.status).to.eq(200)
+                expect(response.body.customers).to.length(limit)
+            })
+        })
+    })
+    
+    context('Request with get industry customers', () => {
+        const industry = ['Logistics', 'Retail', 'HR', 'Logistics']
+        it.each(industry)('Filter to customers with industry', (industry) => {
+            cy.request(`${Cypress.env('baseUrl')}?page=1&industry=${industry}`).then((response) => { 
+                expect(response.status).to.eq(200)
+                expect(response.body.customers[0].industry).to.eq(industry)
+            })
+        })
+    })
+    context('Request with get size customers', () => {
+        const sizes = ['Small','Medium','Enterprise','Large Enterprise','Very Large Enterprise']
+        it.each(sizes)('Filter to customer with size', (sizes) => {
+            cy.request(`${Cypress.env('baseUrl')}?page=1&size=${sizes}&industry=All`).then((response) => { 
+                expect(response.status).to.eq(200)
+                expect(response.body.customers[0].size).to.eq(sizes)
+            })
+        })
+    })
+    
+    context('Bad Requests', () => {
+        it('Check message for invalid page', () => {
+            cy.request({url: `${Cypress.env('baseUrl')}?page=0`, failOnStatusCode: false}).then((response) => { 
```

Comment:
Utilize desestruturação de objetos.

Reply:
Além disso, evite uma linha tão longa, a qual em monitores pequenos pode exigir _scroll_ horizontal.

Segue minha sugestão:

```suggestion
            cy.request({
              url: `${Cypress.env('baseUrl')}?page=0`,
              failOnStatusCode: false
            }).then((response) => { 
```

Reply:
Além da desestruturação da resposta, conforme mencionado pelo @ldasilvaNZ11.